src/Microsoft.ML.Data/Scorers/MultiClassClassifierScorer.cs

```diff
@@ -386,38 +386,30 @@ public void GetMetadata<TValue>(string kind, int col, ref TValue value)
                     }
                 }
 
-                private sealed class RowImpl : Row
+                private sealed class RowImpl : WrappingRow
                 {
-                    private readonly Row _row;
                     private readonly Schema _schema;
 
-                    public override long Batch => _row.Batch;
-                    public override long Position => _row.Position;
                     // The schema is of course the only difference from _row.
                     public override Schema Schema => _schema;
 
                     public RowImpl(Row row, Schema schema)
+                        : base(row)
                     {
                         Contracts.AssertValue(row);
                         Contracts.AssertValue(schema);
 
-                        _row = row;
                         _schema = schema;
                     }
 
                     public override bool IsColumnActive(int col)
                     {
-                        return _row.IsColumnActive(col);
+                        return Input.IsColumnActive(col);
```

Comment:
>return [](start = 24, length = 6)

you tend to prefer => for other one-liners.

Reply:
Kinda. Sometimes.